plugins/generator-1.20.1/forge-1.20.1/block.definition.yaml

```diff
@@ -386,28 +386,22 @@ localizationkeys:
   - key: block.@modid.@registryname
     mapto: name
 
-global_templates:
-  - template: block/mineable_tag.json.ftl
-    writer: json
-    variables: "type=pickaxe"
-    name: "@RESROOT/data/minecraft/tags/blocks/mineable/pickaxe.json"
-    condition: "${w.getGElementsOfType('block')?filter(e -> e.destroyTool.equals('pickaxe'))?size != 0}"
-    hidden: true
-  - template: block/mineable_tag.json.ftl
-    writer: json
-    variables: "type=axe"
-    name: "@RESROOT/data/minecraft/tags/blocks/mineable/axe.json"
-    condition: "${w.getGElementsOfType('block')?filter(e -> e.destroyTool.equals('axe'))?size != 0}"
-    hidden: true
-  - template: block/mineable_tag.json.ftl
-    writer: json
-    variables: "type=shovel"
-    name: "@RESROOT/data/minecraft/tags/blocks/mineable/shovel.json"
-    condition: "${w.getGElementsOfType('block')?filter(e -> e.destroyTool.equals('shovel'))?size != 0}"
-    hidden: true
-  - template: block/mineable_tag.json.ftl
-    writer: json
-    variables: "type=hoe"
-    name: "@RESROOT/data/minecraft/tags/blocks/mineable/hoe.json"
-    condition: "${w.getGElementsOfType('block')?filter(e -> e.destroyTool.equals('hoe'))?size != 0}"
-    hidden: true
+tags:
+  - tag: BLOCKS:minecraft:mineable/pickaxe
+    condition: "${data.destroyTool == 'pickaxe'}"
+  - tag: BLOCKS:minecraft:mineable/axe
+    condition: "${data.destroyTool == 'axe'}"
+  - tag: BLOCKS:minecraft:mineable/shovel
+    condition: "${data.destroyTool == 'shovel'}"
+  - tag: BLOCKS:minecraft:mineable/hoe
+    condition: "${data.destroyTool == 'hoe'}"
+  - tag: BLOCKS:minecraft:stairs

```

Comment:
I tried creating a block with these block bases, but the tag isn't generated

Reply:
Good catch, thanks for providing a solution too! :)